src/constants.rs

```diff
@@ -509,30 +509,26 @@ bitflags! {
         const TRIGGERPREIN = 1 << 25;  // %triggerprein dependency
         const KEYRING	= 1 << 26;
         // bit 27 unused
-        const CONFIG	= 1 << 28;
+        const CONFIG	= 1 << 28;    // config() dependency: TODO: wtf is a config dependency?
```

Comment:
depends on another package providing a specific configuration file, I don't think the TODO is constructive

Reply:
The part that I find confusing is what makes it different from any other "file" dependency, and why not depend on that config file directly rather than `config(rpm-basic)` 1.0-1, and why packages both provide it and also self-require it as seen here.

https://github.com/rpm-rs/rpm/pull/115/files#diff-26e82b76ab9e2c9c188e78b8b38612a335b74fb0295201fbd609a74a45582f93R194

I'd also kinda like to know which line is triggering it, I can figure that out myself but I haven't done so yet.

Reply:
@DemiMarie Do you happen to know what is the intended purpose behind putting `config()` dependencies in both the provides and requires sections?